packages/actor-rdf-parse-html/lib/ActorRdfParseHtml.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ActorRdfParse,
+  ActorRdfParseFixedMediaTypes,
+  IActionRdfParse,
+  IActionRootRdfParse,
+  IActorOutputRootRdfParse,
+  IActorRdfParseFixedMediaTypesArgs,
+  IActorRdfParseOutput,
+  IActorTestRootRdfParse,
+} from "@comunica/bus-rdf-parse";
+import {ActionContext, Actor, IActorTest, Mediator} from "@comunica/core";
+import {Readable} from "stream";
+
+/**
+ * A HTML RDF Parse actor that listens on the 'rdf-parse' bus.
+ *
+ * It is able to extract JSON-LD from HTML files and parse the JSON-LD based RDF serializations
+ * and announce the presence of them by media type.
+ */
+// Gebaseerd op JSON LD parser
+export class ActorRdfParseHtml extends ActorRdfParseFixedMediaTypes {
```

Comment:
As far as I understand, this parser _only_ extracts JSON-LD snippets.
I though the goal was to parse RDFa as well?

Reply:
It is the goal, but so far, only the json-ld snippet is extracted (first thing we need for a project)